src/App.tsx

```diff
@@ -109,17 +108,38 @@ const HomePage = () => {
   );
 };
 
+const BottomBar = () => {
+  return (
+    <Container>
+      <Menu>
+        <Menu.Item>
+          <a href="/">Home</a>
+        </Menu.Item>
+        <Menu.Item>
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLScQJKr1wSd1u2riaM_Fpqf65KaFDiviDkw3oG1I1_S9w3Zh4A/viewform?usp=sf_link">Report a Mistake</a>
+        </Menu.Item>
+        <Menu.Item>
+          <a href="https://docs.google.com/forms/d/e/1FAIpQLSc78OI5QJ2nB82BRuLxHS_LkOQjHS2WVcDUOd48o51PRYTThQ/viewform?usp=sf_link">Request an Image</a>
+        </Menu.Item>
+      </Menu>
+    </Container>
+  );
+}
+
 function App() {
   return (
     <Router>
       <Route path="/i/:id">
         <ImagePage />
+        <BottomBar />
```

Comment:
I think these belong in the page themselves. I read _ImagePage_, for example, as everything represented on that page.